### PR TITLE
[LUM-494] - Fix 502 and TypeError for dfr

### PR DESCRIPTION
**Pull request by Ricardo-Remy**
**Current Issue:**
- To calculate the tvl, we frequently rely on external api to connect to client and get the total allocated token per chain -> rpc-imperator.
- By iterating heavily on it we receive 502 from the server, which causes chaining issues to the child methods (aka getApy for instance).

**Proposed solution**
- Save the total_allocated_token amount in DB and query on it every time we use the DFR service, instead of calling the third party apis.
- The tvl will be calculated by the current price in db and the current allocated token instead.
- The assets.scheduler makes sure to update the db every 5 min.

NB: This solution has been tested on a 1hour set without error.

### CODE REVIEW

File: src/http/controllers/core.controller.ts [hunk 1]
```diff
@@ -176,6 +176,7 @@ export class CoreController {
 
     @MessagePattern('updateMetric')
     async updateMetric(@Payload() data: { name: string; value: number }): Promise<void> {
+        console.log('data', data);
```
Comment (Segfaultd) [inline src/http/controllers/core.controller.ts:179]: Oopsi

File: src/http/responses/asset-info.response.ts [hunk 2]
```diff
@@ -21,6 +21,10 @@ export class AssetInfo {
     @ApiProperty()
     @Expose()
     apy?: number;
+
+    @ApiProperty()
+    @Expose({ name: 'totalAllocatedToken' })
```
Comment (Segfaultd) [inline src/http/responses/asset-info.response.ts:26]: Why is this required?

**Review by Segfaultd — CHANGES_REQUESTED** [review]

File: src/http/controllers/core.controller.ts (hunk 1 above)
Comment (Ricardo-Remy) [reply src/http/controllers/core.controller.ts:179]: my bad fixed.

File: src/http/responses/asset-info.response.ts (hunk 2 above)
Comment (Ricardo-Remy) [reply src/http/responses/asset-info.response.ts:26]: thought of changing the alias name for the exposed property. But true we don't do it for the others in the class. I remove it.
